### ISSUE TITLE
LSD

### ISSUE THREAD

**Issue by ruiiiijiiiiang** [catppuccin/catppuccin#2592]
### Link to repository

https://github.com/ruiiiijiiiiang/lsd

### Screenshots

![Screenshot 2024-11-05 at 11 06 37](https://github.com/user-attachments/assets/494998cc-9a6a-47e8-877c-eaabcaf0ad4f)
![Screenshot 2024-11-05 at 11 07 51](https://github.com/user-attachments/assets/5510d4a4-12f7-4a1c-9ef2-bfc15726c82d)
![Screenshot 2024-11-05 at 11 08 22](https://github.com/user-attachments/assets/9eea60c0-98d4-49c2-b6be-462930165dab)
![Screenshot 2024-11-05 at 11 08 48](https://github.com/user-attachments/assets/a1946515-5a1f-4de9-b11a-af98d5dfc74e)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey 👋,
Sorry for the wait on this.

I had the opportunity to try it out earlier and while I personally find it to be too colourful - I think the colours have been followed quite well from LSD defaults.

I have a few comments about the repository:

1. Please delete the [.gitkeep](https://github.com/ruiiiijiiiiang/lsd/blob/main/assets/.gitkeep) file as that is no longer needed.
2. Instead of adding a [separate FAQ section in the README](https://github.com/ruiiiijiiiiang/lsd/blob/main/README.md#-faq), I would link to LSD's documentation in the `Usage` section so that we aren't repeating ourselves. You can find a previous example of this in [catppuccin/yazi](https://github.com/catppuccin/yazi?tab=readme-ov-file#usage).
3. The screenshots/previews included in the repository look quite... desaturated? I would appreciate if you could take the screenshots again.

In any case, my comments are quite minor with regards to the repository itself so I'm happy to approve for transfer into the organisation and we can resolve those comments afterwards. Please read, ["Transferring a repository owned by your personal account"](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account), if you are unsure about the process of transferring the repository to Catppuccin.

**We'll close this issue once the port has been added to our [ports.yml](https://github.com/catppuccin/catppuccin/blob/main/resources/ports.yml)**.

**Comment by ruiiiijiiiiang** [comment]
@sgoudham Thank you for the feedback. I've made an update to address your concerns. I'm not sure why the previous screenshots came out so weird. I took them again on a different computer and they should look better this time.

**Comment by sgoudham** [comment]
Great, thanks for redoing those screenshots!
I see that you have accepted the invite to the organisation, feel free to transfer the repository over whenever you are free! @ruiiiijiiiiang
